src/App.css

```diff
@@ -1,38 +1,22 @@
-.App {
-  text-align: center;
+.navbar-brand {
+  color: aqua;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
-
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.nav-item {
+  margin: 0 0.5rem;
   display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #61dafb;
 }
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+.material-symbols-outlined {
```

Comment:
lines 11, 18, 19, and 21 can be replaced with the Bootstrap classes "float-end fs-1 p-0 rounded-2". rounded-2 is not identical but is very close to the border-radius value here (0.375rem instead of 0.4rem)